fontir/src/ir.rs

```diff
@@ -746,18 +747,11 @@ pub struct NameBuilder {
     version_minor: u32,
 }
 
-impl Default for NameBuilder {
-    fn default() -> Self {
-        let mut builder = Self {
-            names: Default::default(),
-            name_to_key: Default::default(),
-            version_major: Default::default(),
-            version_minor: Default::default(),
-        };
-        // Based on diffing fontmake Oswald build
-        builder.add(NameId::SUBFAMILY_NAME, "Regular".to_string());
-        builder
-    }
+const STYLE_MAP_STYLE_NAMES: [&str; 4] = ["regular", "bold", "italic", "bold italic"];
```

Comment:
in general for arrays or strings I prefer `static` to `const` (since in theory a `const` is instantiated in each place it is referenced), and so this would probably be a static slice of `&str`, like:

```rust
static STYLE_MAP_STYLE_NAMES: &[&str] = &[..];
```

Reply:
Good catch. I'd have thought to use const as well, does it actually produce a worse result or does the compiler fix it for you?